frontends/web/src/locales/en/app.json

```diff
@@ -352,7 +352,7 @@
   },
   "chart": {
     "dataMissing": "Gathering historical data... stay tuned.",
-    "dataUpdating": "updating data…",
+    "dataUpdating": "updating data...",
```

Comment:
hm, I think it doesn't matter if we do locize-pull next time it will be overwritten, but if you update the PR anyway please change to `…`.

Reply:
Oops, done.